controller/network/network.go

```diff
@@ -119,7 +119,7 @@ func NewNetwork(config Config) (*Network, error) {
 		routerChanged:         make(chan *Router, 16),
 		linkController:        newLinkController(config.GetOptions()),
 		linkChanged:           make(chan *Link, 16),
-		forwardingFaults:      make(chan *ForwardingFaultReport, 16),
+		forwardingFaults:      make(chan struct{}, 1),
```

Comment:
Curious to understand why a buffer of 16 faults and now 1?

Reply:
It's now just a signal, indicating that cleanup should be done. It was running the fault stuff in a goroutine already, so now it just does that first, before signalling.